woocommerce/wc-admin
# Update/variations search method

Fixes #1773 

* Adds `product_search` to variations controller.
* Hooks up `product_search` to variation autocompleters.
* Adds product sku to autocompleter keywords.

### Screenshots
<img width="633" alt="Screen Shot 2019-03-13 at 5 22 57 PM" src="https://user-images.githubusercontent.com/10561050/54267777-c864f480-45b4-11e9-9187-d5cc692b96ea.png">

### Detailed test instructions:

1. Create a variable product with many products and varying skus.
2. Check that the autocompleter options correctly return products by searched name/sku.

@psealock Can you double check and let me know if this works with your data?
